src/test/resources/reference.conf

```diff
@@ -80,21 +80,29 @@ clusterDefaults {
   numberOfPreemptibleWorkers = 0
 }
 
-sam {
```

Comment:
I think the top-level `sam` config might still be used by the `StatusService`. Probably want to keep it.